### DISCUSSION TITLE
Utilizaçao de parte do saldo de eventos anteriores.

### DISCUSSION THREAD

**Discussion by jailsonpj** [apyb/comunidade#162] [Pergunta]
Oi gente, tudo bem?

Pessoas, eu comecei os trabalhos referente ao desenvolvimento da Python Norte 2025 que acontecerá em Belém do Pará.
Nosso primeiro trabalho seria em criar a identidade visual do evento como requisito para no desenvolvimento do Plano de Patrocínio e outras atividades.

Eu encontrei nessa [Discussions](https://github.com/apyb/comunidade/discussions/115) que poderiamos pedir a APyB uma utilização do  seu saldo para adiantar um valor que iremos receber no futuro. Lembrando aqui que em todas as organizações da Python Norte terminamos em saldo bastante positivo no caixa e que foi repassado a associação. 

Eu fiz o orçamento com algumas designs e pode ser visto nessa [Issue](https://github.com/PyNorte/pynorte2025-org/issues/1).

Queria saber em que momento podemos verificar a possibilidade da ajuda da APyB nesse início, visto que esse orçamento contém descontos e tem a validade de até o fim do ano de 2024.

OBS: O Big Kahuna da edição de 2025 da Python Norte sou eu :)

@apyb/diretoria

**Comment by cecivieira** [comment]
Pode sim, Jailson

**Reply by jailsonpj** [reply]
Como posso seguir agora? Onde encontro o formulário para isso? Tentei ir atrás de algum mas não encontrei heueheu

**Reply by cecivieira** [reply]
@jailsonpj pode solicitar um pagamento normal usando o código do evento (pyn2025). Seguindo as orientações do Manual https://apyb.python.org.br/associados/guias/como-organizar-eventos/

**Reply by jailsonpj** [reply]
Blz, Obrigado @cecivieira <3
